### PR TITLE
three seconds is better

### PR DESCRIPTION
**Pull request by jalilovr**
Hello, please apply my changes.
I`m increased smart-blink from 1 sec to 3 sec.
What do you think about this?

### CODE REVIEW

File: Main.txt [hunk 1]
```diff
@@ -2,4 +2,5 @@ This main brunch.
 Test this.
 This first brunch.
 Edit Err1.
-Edit Test1lOCAL.
\ No newline at end of file
+Edit Test1lOCAL.
+Slow-Blink
```
Comment (rustamturnik) [inline Main.txt]: Ok,please write smart blink time, after this i merge you chages in my master branch,

**Review by rustamturnik — CHANGES_REQUESTED** [review]
please correct you changes

File: Main.txt (hunk 1 above)
Comment (jalilovr) [reply Main.txt]: Ok

**Comment by jalilovr** [comment]
Soon i can  push my changes

**Pull request merged**